frontend/packages/data-portal/app/components/Viewer/ViewerPage.tsx

```diff
@@ -148,23 +149,29 @@ function ViewerPage({
   const depositionConfigs = buildDepositionsConfig(run.annotations)
```

Comment:
There is a bug happening that I'm not sure which change it is related to, but If you follow these steps it should reproduce it:
1. Go to RN-18177 and click "view tutorial" from the portal page (so not in the viewer yet).
2. Go through the whole tour, and then click close
3. Click on the annotations tab, it should hide a deposition.
4. Then when you click the deposition, it should make the viewer blank.

Maybe something related to the hash not being ready?

Edit -- you can also do this by doing the whole tour, then when done it sets the URL to  http://localhost:8080/view/runs/18177 -- I think maybe clearing the query param is clearing the whole state?

Reply:
Reverting to the numrla URL handling that we had, we are good. I'll try to use a https://v2.remix.run/docs/utils/replace/ to make use of remix, in case they prefer this solution, but reading the documentation, it will just bring more trouble.